### ISSUE TITLE
Properties.overrideParameters doesn't work as expected

### ISSUE THREAD

**Issue by alexandrnikitin** [typelevel/scalacheck#289]
`Properties.overrideParameters` method gets called but doesn't override parameters.
A piece of code to reproduce:

``` scala
import org.scalacheck.Test.Parameters
import org.scalacheck._

object StringSpecification extends Properties("String") {
  import Prop.forAll

  override def overrideParameters(p: Parameters): Parameters = {
    println("called")
    p.withMinSuccessfulTests(1) // this doesn't affect number of iterations
  }

  property("startsWith") = forAll { (a: String, b: String) =>
    println("iteration")
    (a+b).startsWith(a)
  }

}
```

Related to #234 but it didn't get called before this PR.

**Comment by ljwagerfield** [comment]
Has anyone found a solution / workaround to this?

**Comment by ashawley** [comment]
@ljwagerfield See #360 where this was more recently picked up

**Comment by ashawley** [comment]
Closing this issue.  A fix for overriding the parameters of properties was merged in #522.
